.changeset/angry-days-swim.md

```diff
@@ -0,0 +1,28 @@
+---
+'@commercetools-frontend/application-shell-connectors': patch
+---
+
+There is a new React hook which you can use to access the Project Images rewrite configuration [see documentation](https://docs.commercetools.com/custom-applications/api-reference/commercetools-frontend-application-shell-connectors#project-image-settings).
+
+You would use it like this:
+
+function MyComponent() {
+const { isLoading, imageRegex } = useProjectExtensionImageRegex();
+
+if (isLoading) return <LoadingSpinner />;
+
+return (
+<div>
+<h1>Project images regex: {imageRegex}</h1>
+</div>
+);
+}
+
+function MyApp() {
+return (
+<ProjectExtensionProviderForImageRegex>
+<MyComponent />
+</ProjectExtensionProviderForImageRegex>
+);
+}
+Both GetProjectExtensionImageRegex component and withProjectExtensionImageRegex still exists for backwards compatibility but have been marked as deprecated.
```

Comment:
It seems we lost the format here.

Reply:
Thanks! updated:
https://github.com/commercetools/merchant-center-application-kit/pull/2639/commits/bb14670c3c87f5e584ef469e67de893923c79678